woocommerce/woocommerce
# chore: remove WCS canada post mentions

### All Submissions:






Canada Post live rates is not a feature WCS has for new stores.
Removing the banner for CA-based stores.

### Changes proposed in this Pull Request:



Closes # .

### How to test the changes in this Pull Request:

- Set up a store with a US-based address
- Go to http://localhost:8084/wp-admin/admin.php?page=wc-addons
- Banner should be displayed
- Change the address to be CA-based
- Go to http://localhost:8084/wp-admin/admin.php?page=wc-addons
- Banner should no longer be displayed

### Other information:




### Changelog entry

> Tweak - Remove Canada Post from WCS extensions banner.
